packages/analytics/src/utils/constants.ts

```diff
@@ -17,7 +17,10 @@ export const CONSENT_KEYS = [
   DefaultConsentKeys.MARKETING,
   DefaultConsentKeys.PREFERENCES,
 ];
+
 export const CONSENT_CATEGORIES_PROPERTY = 'consentCategories';
 export const LOAD_INTEGRATION_TRACK_TYPE = 'loadIntegration';
 export const ON_SET_USER_TRACK_TYPE = 'onSetUser';
 export const ANALYTICS_UNIQUE_EVENT_ID = '__uniqueEventId';
+export const ANALYTICS_UNIQUE_EVENT_INTEGRATION_DEFAULT_NAME =
```

Comment:
This name I think it is not correct since this is not a name for the integration but for the uniqueEventId (As I said before, I think the property `ANALYTICS_UNIQUE_EVENT_ID ` refers to the same thing)

Reply:
After looking at the code more thoroughly, I see that this prop is just a rename for the default `__uniqueEventId`. The question I have is, why do we have this indirection? Couldn't we have this instead?

```suggestion
export const ANALYTICS_UNIQUE_EVENT_ID = '__blackoutAnalyticsEventId';
```
Then we just need to use the `ANALYTICS_UNIQUE_EVENT_ID` instead of using `ANALYTICS_UNIQUE_EVENT_INTEGRATION_DEFAULT_NAME `.

Reply:
This __blackoutAnalyticsEventId, appeared for the first time in GA4, and now it was replicated into GTM integration, always mapping from analytics __uniqueEventId field.

As these two integrations use the same property name on their datalayer, I set this const on our analytics package, to represent a default name in this kind of situation.

Reply:
I understand that, I am just highlighting the fact that we do not need 2 variables for this job. We could just change the value of the `ANALYTICS_UNIQUE_EVENT_ID` to `__blackoutAnalyticsEventId` and use only that value throughout all integrations. If another integration needs that value to be sent in another property (like Omnitracking does), it can remap it there to the name of its choosing. By following this suggestion you can then change both the GA4 and GTM integrations code to be like this:

```js
const payload = {
      ...(eventProperties as object),
      [coreUtils.ANALYTICS_UNIQUE_EVENT_ID]: get(
      data,
      `context.event.${coreUtils.ANALYTICS_UNIQUE_EVENT_ID}`,
    ),
      type: data.type,
      event: data.event,
    };

```

And we remove the redundancy.

Reply:
I agree that could be more simple and I've applied your suggestion.